hail/src/main/scala/is/hail/annotations/StagedRegionValueBuilder.scala

```diff
@@ -81,7 +166,7 @@ class StagedRegionValueBuilder private(val mb: MethodBuilder, val typ: PType, va
         startOffset.store(region.allocate(t.alignment, t.byteSize))
     else
       startOffset.store(pOffset)
-    assert(staticIdx == 0)
+    staticIdx = 0
```

Comment:
Could you explain this change?

Reply:
I wanted to be able to reuse the same staged RVB in some of the aggregators, and since we track the struct index statically (kind of a hack, but introduced because all of the various field offsets within were causing a constant overflow in the JVM emitted code for large structs we now do this and just increment the offset with respect to a the previous offset) we need to reset it when we're starting a new struct.